test/rulesets/LinearAlgebra/norm.jl

```diff
@@ -53,15 +79,23 @@
         @testset "rrule" begin
             test_rrule(norm, x)
             x isa Matrix && @testset "$MT" for MT in (Diagonal, UpperTriangular, LowerTriangular)
-                # we don't check inference on older julia versions. Improvements to
-                # inference mean on 1.5+ it works, and that is good enough
                 test_rrule(norm, MT(x); check_inferred=VERSION>=v"1.5")
             end
 
             ȳ = rand_tangent(norm(x))
             @test extern(rrule(norm, zero(x))[2](ȳ)[2]) ≈ zero(x)
             @test rrule(norm, x)[2](Zero())[2] isa Zero
         end
+        ndims(x) > 1 && @testset "non-strided" begin
+            xp = if x isa Matrix
+                view(x, [1,2,3], 1:3)
+            elseif x isa Array{T,3}
+                PermutedDimsArray(x, (1,2,3))
+            end
+            @test !(xp isa StridedArray)
+            test_frule(norm, xp ⊢ rand(T, size(xp)))
+            test_rrule(norm, xp ⊢ rand(T, size(xp)))  # rand_tangent does not work here
```

Comment:
Can an issue be openned on ChainRulesTestUtils for this?

Reply:
Fair, i have thoughts but not a discussion for this PR.
lets add the comment saying we are providing the tangents since these are Ints and so `rand_tangent` assumes they are not differentiable.
and then we can merge